tensornetwork/backends/jax/jax_backend.py

```diff
@@ -44,29 +44,34 @@ def convert_to_tensor(self, tensor: Tensor) -> Tensor:
     return result
 
   def concat(self, values: Tensor, axis: int) -> Tensor:
-    return numpy.concatenate(values, axis)
+    return np.concatenate(values, axis)
 
   def randn(self,
             shape: Tuple[int, ...],
-            dtype: Optional[numpy.dtype] = None,
+            dtype: Optional[np.dtype] = None,
             seed: Optional[int] = None) -> Tensor:
     if not seed:
-      seed = numpy.random.randint(0, 2**63)
+      seed = np.random.randint(0, 2**63)
     key = self.jax.random.PRNGKey(seed)
 
     if not dtype:
-      dtype = self.dtype if self.dtype is not None else numpy.float64
+      dtype = self.dtype if self.dtype is not None else np.dtype(np.float64)
 
     def cmplx_randn(complex_dtype, real_dtype):
+      real_dtype = np.dtype(real_dtype)
+      complex_dtype = np.dtype(complex_dtype)
+
       key_2 = self.jax.random.PRNGKey(seed + 1)
-      return self.jax.random.normal(
-          key, shape,
-          dtype=real_dtype) + complex_dtype(1j) * self.jax.random.normal(
-              key_2, shape, dtype=real_dtype)
 
-    if dtype is self.np.complex128:
+      real_part = self.jax.random.normal(key, shape, dtype=real_dtype)
+      complex_part = self.jax.random.normal(key_2, shape, dtype=real_dtype)
+      unit = (np.complex64(1j) if complex_dtype == np.dtype(np.complex64)
+              else np.complex128(1j))
+      return real_part + unit*complex_part
```

Comment:
Add spaces around the `*`

Reply:
done! strange pylint does not catch this